File: src/main/resources/scripts/engine.java
import org.graalvm.shadowed.org.json.JSONObject;
import org.folio.rest.camunda.utility.ScriptEngineUtility;

public String %s(String inArgs) {
  ScriptEngineUtility scriptEngineUtility = new ScriptEngineUtility();
  JSONObject args = scriptEngineUtility.decodeJson(inArgs);
  JSONObject returnObj = scriptEngineUtility.createJson();
  %s
  return scriptEngineUtility.encodeJson(returnObj);
}

File: src/main/resources/scripts/engine.rb
require 'json'
require 'use org.folio.rest.camunda.utility.ScriptEngineUtility'

def %s(inArgs)
  scriptEngineUtility = ScriptEngineUtility();
  args = scriptEngineUtility.decodeJson(inArgs);
  returnObj = scriptEngineUtility.createJson();
  %s
  return scriptEngineUtility.encodeJson(returnObj)
end